CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+ - Added `SpotifyImplicitGrant` as an auth manager option. It provides
+ user authentication without a client secret but sacrifices the ability
+   to refresh the token without user input. (However, read the class
+   docstring for security advisory.)
+
+## [2.12.1] - 2020-05-28
```

Comment:
2.12.1 has not been released yet and I think we will release your PR as part of 2.13.0

For now just put your changes into [Unreleased]